multimodal/src/autogluon/multimodal/utils/data.py

```diff
@@ -199,6 +202,10 @@ def create_data_processor(
             text_max_len=model_config.max_text_len,
             missing_value_strategy=config.data.document.missing_value_strategy,
         )
+    elif data_type == SEMANTIC_SEGMENTATION_IMG:
+        data_processor = SemanticSegImageProcessor(
+            model=model, model_config=model_config, norm_type=model_config.image_norm
```

Comment:
`model` already has the attribute config, which is the same as `model_config` here. So, using only the `model` can initialize the processor.